### PR TITLE
add ethereum transactions to processor

### PR DESCRIPTION
**Pull request by belopash**

### CODE REVIEW

**Comment by belopash** [comment]
I decided not to implement `decodeTxInput`

File: substrate-processor/src/processor/handlerProcessor.ts [hunk 1]
```diff
@@ -913,6 +968,19 @@ class HandlerRunner<S> extends Runner<S, DataHandlers>{
                             log.debug('end')
                         }
                     }
+                    for (let handler of this.getEthereumTranscationHandlers(handlers, item.call)) {
+                        if (item.call.success || handler.triggerForFailedCalls) {
```
Comment (eldargab) [inline substrate-processor/src/processor/handlerProcessor.ts:972]: Hey, the problem here is that successful execution of substrate call doesn't imply successful evm transaction.

https://github.com/paritytech/frontier/blob/ef75c42e9eb65319fd09ae589e3d965633b2ef51/frame/ethereum/src/lib.rs#L597

So, the processing of EVM transactions should be based on `Ethereum.Executed` event.

**Review by eldargab — CHANGES_REQUESTED** [review]

File: substrate-processor/src/processor/handlerProcessor.ts (hunk 1 above)
Comment (belopash) [reply substrate-processor/src/processor/handlerProcessor.ts:972]: Is it possible to do it on processor side or should I fix it in ingester?

Comment (tmcgroul) [reply substrate-processor/src/processor/handlerProcessor.ts:972]: I came up with two options:
1. A mapping developer should request 'Ethereum.Executed' event itself and check if a transaction is successful
2. The substrate-gateway should annotate `call.evmTxSuccess` field like it does for `event.evmTxHash` (this option rely on network runtime which is error prone)

**Comment by eldargab** [comment]
Closing in favour of #98 . Let's continue work and discussion there.

**Pull request closed without merging**